### ISSUE TITLE
scale bounding tree

### ISSUE THREAD

**Issue by feldman4** [nphollon/collision#2]
Is it possible to add a function like `scale : Vector -> Bounds -> Bounds` or `collideScaled : {a | frame, bounds, scale}` that would let you resize an object's bounding tree faster than recomputing it?

**Comment by nphollon** [comment]
Yeah, that would be useful. I think this might make more sense as an extension to the Frame module in geo3d.
